README.md

```diff
@@ -1,7 +1,17 @@
 # renate-od
 RENATE Open Diagnostics (RENATE-OD)
 
-Responsible developer: Dr. Gergo Pokol, www.reak.bme.hu/pokol
+Responsible developers:
+* [Dr. Gergo Pokol](www.reak.bme.hu/pokol)
+* [Ors Asztalos](https://github.com/asztalosors)
+
```

Comment:
Peter Balazs and Matyas Aradi to be added

Reply:
Done